hieradata/site/dev/role/tang.yaml

```diff
@@ -0,0 +1,9 @@
+---
+profile::core::firewall::firewall:
+  "200 accept tang":
+    proto: "tcp"
+    state: "NEW"
+    ipset: "dev src"
+    dport: "7500"
+    action: "accept"
+    require: "Ipset::Set[dev]"
```

Comment:
At a previous ISP we had a collector that ensured all ipsets are configured before the firewall rules. With that single line of code you could get rid of all those `require` statements.

Reply:
That is a very good point... I have no idea why I didn't do that... I will update it.